packages/elements-dev-portal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-dev-portal",
-  "version": "1.7.3",
+  "version": "1.7.4",
   "description": "UI components for composing beautiful developer documentation.",
   "keywords": [],
   "sideEffects": [
```

Comment:
Line 62: does elements-core version need bumping here?

Reply:
good catch - I have bumped the version on line 62!